### ISSUE TITLE
Notifications paged by date is cumbersome

### ISSUE THREAD

**Issue by Giszmo** [hzrd149/nostrudel#153]
The new feature where a date filter for notifications was introduced, makes visiting the notifications so much more cumbersome. Please revert.

When using nostrudel around midnight and suddenly the list is just empty, when using it with a secondary, low traffic account and you have to page back a week to find notifications and just generally when you are not an influencer and therefore see just one or two notifications per day, the new layout is just awful. :heart:

**Comment by 0xtrr** [comment]
I agree on this one. What about loading the latest 24 hours and on scroll, load another 24 hours? Not sure if it's a good idea but just an idea.

**Comment by Giszmo** [comment]
24h windows are mostly empty on my less used account. I suspect that the current concept was tested with a mildly active account. With a highly active account you would see 10 minutes worth of notifications per page and on low traffic accounts it's a month of notifications per page.

For me the most important aspect is to not show an empty page prior to scrolling or hitting buttons. Please always show something.

**Comment by 0xtrr** [comment]
I also see that a lot of notifications doesn't show up in the new notification window. The first image is from the "next" version (deployed yesterday) and the second image is from the latest stable release (0.39.0).
![next version](https://github.com/user-attachments/assets/9990301b-5e48-4135-803e-cc770715e0d7)

![latest release](https://github.com/user-attachments/assets/3781ed66-21ed-4e57-a2cd-e8081275f42f)

**Comment by Giszmo** [comment]
I very often miss notifications and have to reload the page to get them, so I'm not sure this really is a difference between next and stable.
